From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: hexchain <i@hexchain.org>
Date: Sun, 5 Sep 2021 08:31:13 +0000
Subject: [PATCH] Fix FIND_LLVM_PROG arguments after 6d6edb1bbb

(cherry picked from commit bf9a0f7ef45fdd442ea45065f612f19cecb67cf4)
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index c6a1c96d02..cf93f10938 100644
--- a/configure.ac
+++ b/configure.ac
@@ -735,14 +735,14 @@ AC_SUBST([ClangCmd])
 dnl ** Which LLVM llc to use?
 dnl --------------------------------------------------------------
 AC_ARG_VAR(LLC,[Use as the path to LLVM's llc [default=autodetect]])
-FIND_LLVM_PROG([LLC], [llc], [$LlvmVersion])
+FIND_LLVM_PROG([LLC], [llc], [$LlvmMinVersion], [$LlvmMaxVersion])
 LlcCmd="$LLC"
 AC_SUBST([LlcCmd])
 
 dnl ** Which LLVM opt to use?
 dnl --------------------------------------------------------------
 AC_ARG_VAR(OPT,[Use as the path to LLVM's opt [default=autodetect]])
-FIND_LLVM_PROG([OPT], [opt], [$LlvmVersion])
+FIND_LLVM_PROG([OPT], [opt], [$LlvmMinVersion], [$LlvmMaxVersion])
 OptCmd="$OPT"
 AC_SUBST([OptCmd])
 
-- 
2.33.1

